src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -15,6 +15,22 @@ class CNATableComponent extends MSKTable<DiscreteCopyNumberData> {
 
 type CNATableColumn = Column<DiscreteCopyNumberData>&{order:number};
 
+export enum AlterationTypes {
+    'DeepDel' = -2,
+    'AMP'= 2
+};
+
+export function renderAlterationTypes(value: number) {
```

Comment:
Should we create a `CNAColumnFormatter` class, and move the `renderAlterationTypes` function there, as well as the `AlterationTypes` enum?